server/middleware/auditMiddleware.ts

```diff
@@ -1,4 +1,5 @@
 import type { NextFunction, Request, RequestHandler, Response } from 'express'
+import { type Key, pathToRegexp } from 'path-to-regexp'
```

Comment:
This is what I was talking about the other day. This should really be two lines, one for the type and one for the non-type import, to fit our style. Should probably work out if we can get ESLint to enforce this (again?)